ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2362,6 +2362,104 @@
       "mediaType": "image/svg+xml"
     }
   },
+  {
+    "id": "nodejs-postgres",
+    "creator": "Ibrahim Jarif",
+    "name": "Nodejs and Postgres Stack",
+    "description": "Nodejs stack with Postgres",
+    "scope": "general",
```

Comment:
hello @jarifibrahim , AFAIK it is more an `advanced` stack (for the scope)

Reply:
Sure. I'll change this. :)

Reply:
I've changed it in https://github.com/eclipse/che/pull/12104/commits/d2c468082c83d0d15f6ffbab83cca177630b69c6